business/workloads.go

```diff
@@ -458,8 +458,10 @@ func (in *WorkloadService) GetWorkload(ctx context.Context, criteria WorkloadCri
 	go func() {
 		defer wg.Done()
 		conf := in.config
-		app := workload.Labels[conf.IstioLabels.AppLabelName]
-		version := workload.Labels[conf.IstioLabels.VersionLabelName]
+		appLabelName, _ := conf.GetAppLabelName(workload.Labels)
+		verLabelName, _ := conf.GetVersionLabelName(workload.Labels)
```

Comment:
are we sure these are always going to be found? I see the original code didn't care; perhaps we take this opportunity to log this (even if at trace level) when app/version labels are not found?

Reply:
it looks like the downstream code is ok with app and/or version being `""`, so I think it's fine as is.